spec/index.html

```diff
@@ -1021,7 +942,7 @@ <h4 id="grant"><a class="toc-anchor" href="#grant" >§</a> Grant</h4>
 <li>The message object <strong><strong>MUST</strong></strong> contain a <code>descriptor</code> property, and its value <strong><strong>MUST</strong></strong> be a JSON object composed as follows:
 <ul>
 <li>The object <strong><strong>MUST</strong></strong> contain a <code>method</code> property, and its value <strong><strong>MUST</strong></strong> be the string <code>PermissionsGrant</code>.</li>
-<li>The object <strong><strong>MUST</strong></strong> contain an <code>permissionGrantId</code> property, and its value <strong><strong>MUST</strong></strong> be a [<a class="spec-reference" href="#ref:RFC4122">RFC4122</a>] UUID Version 4 string representing the reply object.</li>
+<li>The object <strong><strong>MUST</strong></strong> contain a <code>permissionGrantId</code> property, and its value <strong><strong>MUST</strong></strong> be a [<a class="spec-reference" href="#ref:RFC4122">RFC4122</a>] UUID Version 4 string representing the reply object.</li>
```

Comment:
Once more, `a RFC4122` vs `an RFC4122`... I think the latter is correct.
```suggestion
<li>The object <strong><strong>MUST</strong></strong> contain a <code>permissionGrantId</code> property, and its value <strong><strong>MUST</strong></strong> be an [<a class="spec-reference" href="#ref:RFC4122">RFC4122</a>] UUID Version 4 string representing the reply object.</li>
```

Reply:
There are a bunch more of these. I suggest you do a global search-and-replace from `be a [<a class="spec-reference" href="#ref:RFC4122` to `be an [<a class="spec-reference" href="#ref:RFC4122`